holoviews/core/spaces.py

```diff
@@ -429,6 +439,10 @@ class Callable(param.Parameterized):
          The list of inputs the callable function is wrapping. Used
          to allow deep access to streams in chained Callables.""")
 
+    link_inputs = param.Boolean(default=True, doc="""
+         Whether a plot created from this Callable should be linked
+         to the declared inputs.""")
```

Comment:
Similar issue with the ``link_inputs`` parameter on operations - it needs to explain a lot more but in terms of ``Callable`` concepts this time.